addons/calendar/tests/test_event_recurrence.py

```diff
@@ -469,6 +469,11 @@ def test_change_week_day_rrule(self):
         self.assertFalse(self.recurrence.tue)
         self.assertTrue(self.recurrence.wed)
 
+    def test_rrule_x_params(self):
+        self.recurrence.rrule = 'RRULE;X-EVOLUTION-ENDDATE=20191112:FREQ=WEEKLY;COUNT=3;X-MAIL-special=1;BYDAY=WE;X-RELATIVE=True' # from TU to WE
```

Comment:
I didn't have a test before in `calendar` because no test added the `RRULE:` prefix. However, after seeing that it works fine too, I added this test.